Packs/PcapAnalysis/Scripts/PcapFileExtractor/PcapFileExtractor.py

```diff
@@ -1,64 +1,155 @@
-from typing import Dict, Union
+import hashlib
+import subprocess
+import tempfile
+from typing import List, Optional, Set, Union
 
-import demistomock as demisto
+import magic
 from CommonServerPython import *  # noqa: E402 lgtm [py/polluting-import]
-from tempfile import mkdtemp
-import shutil
-import subprocess
-import hashlib
+
+import demistomock as demisto
 
 
-def get_pcap_path(args: Dict) -> str:
-    """
+class InclusiveExclusive:
+    INCLUSIVE: str = 'inclusive'
+    EXCLUSIVE: str = 'exclusive'
 
-    Args:
-        args: argument from demisto
 
-    Returns: path of pcap file
+def get_file_path_from_id(entry_id: Optional[str] = None):
```

Comment:
with regards to my comments about that func in `main`, please add return type hint